### PR TITLE
chore: add guidance on getters/setters to coding standards

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

**Comment by jelbourn** [comment]
@amcdnl added

File: CODING_STANDARDS.md [hunk 1]
```diff
@@ -167,6 +167,14 @@ be part of the user-facing API. This typically applies to symbols used in templa
 Additionally, the `@docs-private` JsDoc annotation can be used to hide any symbol from the public
 API docs.
 
+
+#### Getters and Setters
+* Always use a `readonly` property instead of a getter (with no setter) when possible.
```
Comment (andrewseguin) [inline CODING_STANDARDS.md]: Can you give an example? My brain is struggling to parse the use case

File: CODING_STANDARDS.md [hunk 2]
```diff
@@ -167,6 +167,14 @@ be part of the user-facing API. This typically applies to symbols used in templa
...
+* Avoid long getters and setters. If the logic of an accessor would take more than three lines,
+introduce a new method to contain the logic.
+* A getter should also immediately precede its corresponding setter.
```
Comment (andrewseguin) [inline CODING_STANDARDS.md]: Remove "also"

Comment (andrewseguin) [inline CODING_STANDARDS.md]: If `@Input` is on setter, it should come first right? This is saying that getters precede the setter (getters come first)

File: CODING_STANDARDS.md (hunk 1 above)
Comment (jelbourn) [reply CODING_STANDARDS.md]: Added

File: CODING_STANDARDS.md (hunk 2 above)
Comment (jelbourn) [reply CODING_STANDARDS.md]: Done

Comment (jelbourn) [reply CODING_STANDARDS.md]: Getters come first, `@input` always goes on the getter.

**Comment by jelbourn** [comment]
@amcdnl @andrewseguin poke for review

**Pull request merged**